cmd/kube-proxy/app/server.go

```diff
@@ -233,10 +233,8 @@ func (o *Options) Complete() error {
 		return err
 	}
 
-	if err := utilfeature.DefaultMutableFeatureGate.SetFromMap(o.config.FeatureGates); err != nil {
-		return err
-	}
-	return nil
+	err := utilfeature.DefaultMutableFeatureGate.SetFromMap(o.config.FeatureGates)
+	return err
```

Comment:
why not `return utilfeature....` everywhere?

Reply:
@apelisse updated PTAL